djmoney/models/fields.py

```diff
@@ -196,25 +196,34 @@ def __init__(
         Field.creation_counter += 1
 
     def setup_default(self, default, default_currency, nullable):
-        if isinstance(default, (str, bytes)):
+        if default in (None, NOT_PROVIDED) or isinstance(default, Money):
```

Comment:
Should it be `MONEY_CLASSES` in `isinstance` as in other places above?

Reply:
I don't think so, as a bit further down we'll convert `OldMoney -> Money`

https://github.com/flaeppe/django-money/blob/e7d558c9cdf92227e997ef65fb6f1d3cdf87c14b/djmoney/models/fields.py#L216-L217